gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,86 @@
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in PDK. The following docs do not include steps that occur outside of PDK, such as xref:policies-pdk-deploy-policies.adoc[applying a policy].
+
+After you xref:policies-pdk-create-project.adoc[Create a New Project], you might iteratively complete some steps. For example, while writing the policy's source code, you may need to return to xref:policies-pdk-create-schema-definition.adoc[Define Policy Configuration Parameters] to add additional parameters, or you might need to recompile the policy after debugging the source code.
+
+The commands associated with each step are provided for reference. However, configuration is necessary before running some of the commands.
+
+To get started creating a custom policy with PDK, follow the suggested steps:
+
+. xref:policies-pdk-create-project.adoc[Create a New Project] 
++
+[source,ssh]
+----
+anypoint-cli-v4 pdk policy-project create --name <my-custom-policy>
+----
+
+. xref:policies-pdk-create-schema-definition.adoc[Define a Schema Definitions]
++
+Modify the `definition/gcl.yaml` file to define metadata and configuration parameters for your custom policy.
++
+Run the `build-asset-files` throughout the development process to update the `src/config.rs` file with your new configuration parameters:
++
+[source,ssh]
+----
+make build-asset-files
+----
+
+. xref:policies-pdk-configure-features.adoc[Configure Custom Policy Features]
++
+Implement your custom logic in the `src/lib.rs` file.
+
+. xref:policies-pdk-compile-policies.adoc[Compile Custom a Custom Policy]
++
+Compiling your policy creates the `WebAssembly` binary file and configuration files necessary to upload your policy to Exchange and to deploy it to a Flex Gateway.
++
+Run the `build` command to compile your policy:
++
+[source,ssh]
+----
+make build
+----
+
+. xref:policies-pdk-debug-local.adoc[Debug Your Policy Locally with PDK]
++
+Use the provided Flex Gateway configuration files to debug your policy locally with PDK before distributing it on Exchange.
++
+After you complete the steps in the tutorial, run the `run` command to begin testing locally:
++
+[source,ssh]
+----
+make run
+----
+
+. xref:policies-pdk-publish-policies.adoc#publish[Publish Your Policy on Exchange]
++
+After testing your policy locally, publish your policy on Exchange to distribute and test your policy with your Flex Gateway configuration.
++
+Run the `publish` command to publish your policy:
++
+[source,ssh]
+----
+make publish
+----
++
+After you publish your custom policy, you can test your policy by xref:policies-pdk-deploy-policies.adoc[applying it to an API instance] deployed on Flex Gateway. 
+
+. xref:policies-pdk-publish-policies.adoc#release[Release Your Policy on Exchange]
++
+After testing is complete, release your policy on Exchange to distribute and test your policy.
```

Comment:
```suggestion
Release your policy on Exchange to distribute and test your policy.
```

Reply:
Releasing on Exchange distributes the policy but does not test it. "...make it available for testing"?